packages/cardano-services/test/Program/services/ogmios.test.ts

```diff
@@ -119,9 +118,9 @@ describe.skip('Service dependency abstractions', () => {
           port = await getPort();
           apiUrlBase = `http://localhost:${port}${services.txSubmit.versionPath}/${services.txSubmit.name}`;
           config = { listen: { port } };
-          txSubmitProvider = await getOgmiosTxSubmitProvider(dnsResolver, logger, {
-            ogmiosSrvServiceName: process.env.OGMIOS_SRV_SERVICE_NAME
-          });
+          // txSubmitProvider = await getOgmiosTxSubmitProvider(dnsResolver, logger, {
```

Comment:
Should this be removed or is it here until the tx submit worker has been migrated?

Reply:
These tests are all `describe.skip`ed in `conway-era`. There is a separate ticket to rework/re-enable those tests, and I just did some minimal changes here so that it compiles